src/runtime/runtime-test.js

```diff
@@ -678,6 +677,15 @@ describes.realWin('Runtime', {}, env => {
       expect(stub).to.be.calledOnce.calledWithExactly(options, callback);
     });
 
+    it('should delegate "showWaitingIndicator"', () => {
+      configuredRuntimeMock.expects('showWaitingIndicator').once()
+          .returns(Promise.resolve());
```

Comment:
Like with the previous issue with return values: let's always test that the value returned here is the exact value returned by the configuredRuntime. An easy way is to return some random value here in the mock, e.g. `11` and then test that the result of `showWaitingIndicator.then(value => {...})` below is equal to the same value (`11`).

Reply:
done